common/models/user.js

```diff
@@ -752,10 +762,10 @@ module.exports = function(User) {
       {
         description: 'Logout a user with access token.',
         accepts: [
-          {arg: 'access_token', type: 'string', required: true, http: function(ctx) {
+          {arg: 'access_token', type: 'string', http: function(ctx) {
             var req = ctx && ctx.req;
             var accessToken = req && req.accessToken;
-            var tokenID = accessToken && accessToken.id;
+            var tokenID = accessToken ? accessToken.id : undefined;
```

Comment:
The original code returned `null` when there was no token provided. strong-remoting does not accept `null` as a `string` value, therefore logout with no access token failed with 400 "missing required argument".